src/styles/container.scss

```diff
@@ -0,0 +1,26 @@
+.container {
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+
+  // &::before {
```

Comment:
You have commented out code starting from this line. If this code is not necessary, then it would be best to remove it to avoid confusion and keep the codebase clean.